katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -0,0 +1,141 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name katello
+# %%global prever .rc1
+%global mainver 3.6.0
+%global release 1.nightly
+
+%global scl_ruby_bin /usr/bin/%{?scl:%{scl_prefix}}ruby
```

Comment:
I wonder if this can't inherit from foreman macros.

Reply:
One way to find out.